### PR TITLE
More Changes for api review

### PR DESCRIPTION
**Pull request by simplynaveen20**
More Changes for api review

### CODE REVIEW

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/ChangeFeedEncryptionProcessorBuilder.java [hunk 1]
```diff
@@ -21,7 +21,7 @@
  * Helper class to build a encryption supported {@link ChangeFeedProcessor} instance.
  *
  */
-public class ChangeFeedEncryptionProcessorBuilder {
+public final class ChangeFeedEncryptionProcessorBuilder {
```
Comment (FabianMeiswinkel) [inline sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/ChangeFeedEncryptionProcessorBuilder.java:24]: IMO these should only be final where they are final in the Core SDK for consistency reasons?

Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/ChangeFeedEncryptionProcessorBuilder.java:24]: Seems weird that ChangeFeedEncryptionProcessorBuilder is final but ChangeFeedProcessorBuilder is not?

Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/ChangeFeedEncryptionProcessorBuilder.java:24]: Yea we discussed that, we should do final when we know we don't want cx to extend it. In cosmos core there is inconsistency  and no actual reason why we didn't do it. In future if use case comes we can open it again ,as it wont be breaking change

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncContainer.java [hunk 2]
```diff
@@ -955,8 +953,6 @@ ItemDeserializer getItemDeserializer() {
      * Use {@link CosmosBatchResponse#isSuccessStatusCode} on the response returned to ensure that the
      * transactional batch succeeded.
      */
-    @Beta(value = Beta.SinceVersion.V1, warningText =
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncContainer.java:958]: why we remove the beta tag here?

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncContainer.java (hunk 2 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/CosmosEncryptionAsyncContainer.java:958]: yes

**Review by kushagraThapar — APPROVED** [review]
LGTM, thanks @simplynaveen20

File: sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/models/KeyEncryptionAlgorithm.java [hunk 3]
```diff
@@ -11,16 +11,16 @@
  * Represents the encryption algorithms supported for key encryption.
  *
  */
-public enum KeyEncryptionKeyAlgorithm {
+public enum KeyEncryptionAlgorithm {
```
[the remark this answers is not in the document]
Comment (simplynaveen20) [reply sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/models/KeyEncryptionAlgorithm.java:14]: Yes I am avoiding any code change in shaded code

**Pull request merged**